handlers/officersHandler_test.go

```diff
@@ -0,0 +1,93 @@
+package handlers
+
+import (
+	"bytes"
+	. "github.com/smartystreets/goconvey/convey"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+const (
+	requestBody = `{
+        "officers" : [
+			{
+				"company_number" : "99999999",
+				"company_name" : "Test company 1",
+				"forename" : "Test forename 1",
+				"middle_name" : "Test middle name 1",
+				"surname" : "Test surname 1",
+				"age" : 25
+			},
+			{
+				"company_number" : "55555555",
+				"company_name" : "Test company 2",
+				"forename" : "Test forename 2",
+				"middle_name" : "Test middle name 2",
+				"surname" : "Test surname 2",
+				"age" : 39
+			}
+    	],
+    	"created_at" : "01/01/2000",
```

Comment:
The delta should have createdTime

Reply:
Will update to CreatedTime as that's what comes out of the DB package.